### DISCUSSION TITLE
How to change CORE TEXT FORMAT & CONTENT TYPES default to MARKDOWN instead of STANDARD plain text?

### DISCUSSION THREAD

**Discussion by Damianus500** [freeplane/freeplane#646] [Help] [answered]
Just re-discovered Freeplane. 

I am using Pop OS 22.04 and happy to find a LINUX mindmap software in active development.

I use Markdown extensively and do not want to change the FORMAT for every new node created.

How to change CORE TEXT FORMAT & CONTENT TYPES default to MARKDOWN instead of STANDARD plain text?

Any advice much appreciated!

Damianus

**Comment by macmarrum** [comment, marked as answer]
Hi Damianus

That's great. Welcome back to Freeplane.

To have Format of each node core, details and note as Markdown, you can set each of them on the `Default` Style → `Format->Manage Styles->Edit styles`

More about styles: https://docs.freeplane.org/user-documentation/styles.html

You might also be interested in https://docs.freeplane.org/user-documentation/css-for-html-or-markdown.html

Mac
